cmd/harbor/root/retention/cmd.go

```diff
@@ -0,0 +1,21 @@
+package retention
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func Retention() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "retention",
+		Short:   "Manage retention rule in the project",
+		Long:    `Manage retention rules in the project in Harbor`,
```

Comment:
We can use the Long explanation to explain what does this command do. 
Since retention is a TAG RETENTION policy, we should communicate this well to the user.

Reply:
Also mention, "user can only create 15 tag retention rules per project".